src/theme/DocSidebar/styles.scss

```diff
@@ -0,0 +1,67 @@
+@import "../../../src/css/index";
+
+.menu {
+  padding: 0;
+
+  .menu__list {
+    &-item {
+      margin: 0;
+    }
+
+    .menu__link {
+      color: map-get($monochrome, black);
+      font-weight: $normal;
+      font-size: 14px;
+      line-height: 24px;
+      padding: 8px 16px;
+
+      &--active {
+        background: map-get($monochrome, white);
+        font-weight: $bold;
+      }
+
+      &:hover {
+        background: map-get($monochrome, light-tint);
+      }
+    }
+
+    .menu__list {
+      margin-left: 0;
+      overflow-y: hidden;
```

Comment:
What's the reason behind adding that?

Reply:
We can see scroll because it has transition